js/whitebit.js

```diff
@@ -503,42 +572,65 @@ module.exports = class whitebit extends Exchange {
         //         "isBuyerMaker":false
         //     }
         //
+        // orderTrades (v4Private)
+        //       {
+        //          "time": 1593342324.613711,      // Timestamp of executed order
+        //          "fee": "0.00000419198",         // fee that you pay
```

Comment:
We should add the fee parsing below, see how that is done in `parseTrade` implementations in the other exchanges.